.github/workflows/branch-cicd.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
 
         strategy:
             matrix:
-                java-version: [8, 11]
+                java-version: [11, 18]
```

Comment:
@al-niessner we are used to only support JDK long term support versions (8, 11, 17), is it easy to switch here from 18 to 17 ? That is important since otherwise our users might be required to install multiple jdk as a requirements for our tools. Thanks

Reply:
@tloubrieu-jpl @al-niessner I did this here: https://github.com/NASA-PDS/validate/pull/600/commits/9024b2265a45e5163a9e5ea6972d31dceb40302a.

The software does not build in Java 8 anymore with the OpenSearch library added as a dependency. I don't think we need that anymore. I just put java 18 in there since it is latest, but we can change that to 17. it is just for testing purposes, not for building.